src/components/checkbox/checkbox-group.html.twig

```diff
@@ -4,27 +4,27 @@
   Parameters:
   - "items" (array) (default: []): array of checkbox_button
   - "name" (string) (default: '')
-  - "invalid" (boolean) (default: false)
-  - "invalid_icon" (object of type "icon") (default: {})
-  - "sr_invalid_icon" (string) (default: ''): additional label for the invalid icon; for screen readers
+  - "feedback_type" (string) (default: ''): type of feedback message; can be "error", "success", "warning"
+  - "feedback_icon" (object of type "icon") (default: {}): icon for the feedback message
+  - "sr_feedback_icon" (string) (default: ''): additional label for the feedback icon; for screen readers
   - "required" (boolean) (default: false)
   - "extra_classes" (string) (default: '')
   - "extra_attributes" (optional) (array) (default: [])
     - "name" (string) Attribute name, eg. 'data-test'
     - "value" (optional) (string) Attribute value, eg: 'data-test-1'
   Blocks:
-  - "invalid_text"
+  - "feedback_text"
 #}
 
 {# Internal properties #}
 
 {% set _items = items|default([]) %}
 {% set _name = name|default('') %}
-{% set _invalid = invalid|default(false) %}
-{% set _invalid_icon = invalid_icon|default({}) %}
-{% set _sr_invalid_icon = sr_invalid_icon|default('') %}
+{% set _feedback_type = feedback_type|default('') %}
```

Comment:
i would keep a boolean parameter to "activate" the feedback, to keep consistency with what we used to have, otherwise the users would have to completely change their approach if they were sending the parameter by default and relying on the boolean to make them visible or not.
At this point i would also set a default for the feedback_type "error", in the end this will be the basic usage if any other will ever be used, so this way we make the transition easier, the parameter names will change but their logic will not.